CHANGELOG.md

```diff
@@ -7,63 +7,72 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 NOTE: As semantic versioning states all 0.y.z releases can contain breaking changes in API (flags, grpc API, any backward compatibility)
 
-We use *breaking* word for marking changes that are not backward compatible (relates only to v0.y.z releases.)
+We use *breaking :warning:* word for marking changes that are not backward compatible (relates only to v0.y.z releases.)
 
 ## Unreleased
 
-## [v0.15.0](https://github.com/thanos-io/thanos/releases) - in release process.
-
-:warning: **WARNING** :warning: Thanos Rule's `/api/v1/rules` endpoint no longer returns the old, deprecated `partial_response_strategy`. The old, deprecated value has been fixed to `WARN` for quite some time. _Please_ use `partialResponseStrategy`.
-
-:warning: **WARNING** :warning: The `sse_encryption` value is now deprecated in favour of `sse_config`. If you used `sse_encryption`, the migration strategy is to set up the following block:
-
-```yaml
-
----
-sse_config:
-  type: SSE-S3
-```
+## [v0.15.0](https://github.com/thanos-io/thanos/releases) - 2020.09.07
 
+Highlights:
+* Added new Thanos component: [Query Frontend](https://thanos.io/tip/components/query-frontend/) responsible for response caching,
+query scheduling and parallelization (based on Cortex Query Frontend).
+* Added various new, improved UIs to Thanos based on React: Querier' BuildInfo & Flags, Ruler UI, BlockViewer.
+* Optimized Sidecar, Store, Receive, Ruler data retrieval with new TSDB ChunkIterator, capping chunks to 120 samples, fixed various leaks.
```

Comment:
`fixed various leaks` sounds like a separate bullet point. If it is related to the TSBD ChunkIterator, then it should be `, which fixed various leaks.`